lib/prepare-system.sh

```diff
@@ -5,6 +5,15 @@ source lib/functions.sh
 
 log "INFO" "Preparing your system for bridgehead installation ..."
 
+# Check, if running in WSL
+if [[ $(grep -i Microsoft /proc/version) ]]; then
```

Comment:
What will happen if this has error code != 0?

Reply:
According to the grep documentation, grep returns only 0, 1, and 2. 0 and 1 have the exact semantics we want in this comparison, 2 indicates an error and would lead to an exit(1) of the install script. While exiting is certainly a sensible approach (e.g. on Macs, that don't implement /proc/*), do you think we need more error handling here, @lablans?

Reply:
My concern is that the whole bash script might crash if a non-zero return value is given here (as requested by the `-e` flag). Again, I don't know about bash to say for sure, just wanted to prevent an error. If you are sure this won't trigger `-e`, go ahead with the merge.